src/actions/pause.js

```diff
@@ -225,3 +235,35 @@ export function loadObjectProperties(object: any) {
     });
   };
 }
+
+/**
+ * @memberOf actions/pause
+ * @static
+ * @param stepType
+ * @returns {function(ThunkArgs)}
+ */
+export function astCommand(stepType: string) {
```

Comment:
I called it astCommand, because it does similar things to command action. but it tries to use the parser AST to evaluate which command should be executed.